sources/packages/backend/apps/db-migrations/src/sql/DynamicFormConfigurations/Insert-dynamic-form-configurations-2026-2027.sql

```diff
@@ -0,0 +1,77 @@
+-- Insert configurations for student financial aid application form type.
+INSERT INTO
+    sims.dynamic_form_configurations (
+        form_type,
+        program_year_id,
+        offering_intensity,
+        form_definition_name
+    )
+VALUES
+    (
+        'Student financial aid application',
+        (
+            SELECT
+                id
+            FROM
+                sims.program_years
+            WHERE
+                program_year = '2026-2027'
+        ),
+        'Full Time',
+        'SFAA2026-27-FT'
```

Comment:
I know that existing DB values use the form name(SFAA2025-26-FT) instead of the form path(sfaa2025-26-ft). It is better to start using the form path here instead.